api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1118 @@
+          payload of events of each type. This ultimate format the payload must adhere to is reflected in
+          the `effective_schema`. See description of field `category` for more details of its generation.
+
+          This field is informative and its contents cannot be manipulated directly.
+
+      validation_strategies:
+        description: |
+          Determines the validation that has to be executed upon reception of Events of this type. Events are
+          rejected if any of the rules fail (see details of Problem response on the Event publishing
+          methods).
+
+          Rule evaluation order is the same as in this array.
+
+          If not explicitly set, default value will respect the definition of the `EventType.category`.
+        type: array
+        items:
+          $ref: '#/definitions/EventValidationStrategy'
+
+      enrichment_strategies:
+        description: |
+          Determines the enrichment to be performed on an Event upon reception. Enrichment is performed once
+          upon reception (and after validation) of an Event and is only possible on fields that are not
+          defined on the incoming Event.
+
+          See documentation for the write operation for details on behaviour in case of unsuccessful
+          enrichment.
+        type: array
+        items:
+          $ref: '#/definitions/EventEnrichmentStrategy'
+        default: [{name: 'metadata-enrichment'}]
+
+      partition_resolution_strategy:
+        description: |
+          Determines how the assignment of the event to a Partition should be handled.
+
+        type: object
+        $ref: '#/definitions/PartitionResolutionStrategy'
+
+      schema:
+        type: object
+        $ref: '#/definitions/EventTypeSchema'
+        description: |
+          The schema for this EventType. This is expected to be a json schema in yaml format (other formats
+          might be added in the future).
+
+      data_key_fields:
+        type: array
+        items:
+          type: string
+        description: |
+          Indicators of the path of the properties that constitute the primary key (identifier) of the data
+          within this Event.
+          If set MUST be a valid required field as defined in the schema.
+          (TBD should be required? Is applicable only to both Business and DataChange Events?)
+
+      ordering_key_field:
+        type: string
+        description: |
+          Indicator of the field used for guaranteeing the ordering of Events of this type (used by the
+          PartitionResolutionStrategy).
+          If set MUST be a valid required field as defined in the schema.
+
+    required:
+      - name
+      - category
+
+  EventTypeSchema:
+    properties:
+      type:
+        type: string
+        example: 'json-schema'
+        description: The type of schema definition (avro, json-schema, etc).
+      schema:
```

Comment:
I think it's a bit unfortunate to have `EventType.schema.schema`, i.e. an object `Schema` that exposes a property `schema`.